measurement_scripts/tag_measurement.py

```diff
@@ -0,0 +1,120 @@
+from measurement_exception import MeasurementScriptBug
+from tqdm import tqdm
+
+TEST_DATA = os.getenv("TAG_TEST_DATA")
+TEST_URL = os.getenv("TAG_URL")
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s.%(msecs)03d %(levelname)s: %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def create_true_pred_list(
+    true_tags: List[Dict[str, Any]],
+    pred_tags: List[Dict[str, Any]],
+) -> Tuple[List[int], List[int]]:
+    """
+    Creates list of true and prediction labels with 1s and 0s.
+
+    Args:
+        true_tags (List[Dict[str, Any]]): True tags, ground truth values
+        pred_tags (List[Dict[str, Any]]): Prediction tags, given by model
+
+    Returns:
+        Tuple[List[int], List[int]]: True and prediction labels
+    """
+    all_tags = true_tags.copy()
+    for pred in pred_tags:
+        for i, tag in enumerate(all_tags):
+            if pred["name"] == tag["name"]:
+                if pred["count"] != tag["count"]:
+                    all_tags[i]["count"] = max(pred["count"], tag["count"])
+                break
+        else:
+            all_tags.append(pred)
+
+    true_labels = []
+    pred_labels = []
+    for tag in all_tags:
+        for pred in pred_tags:
+            if pred["name"] == tag["name"]:
+                if pred["count"] == tag["count"]:
+                    pred_labels.extend([1] * tag["count"])
+                else:
+                    pred_labels.extend([1] * pred["count"])
+                    pred_labels.extend([0] * (tag["count"] - pred["count"]))
+                break
+        else:
+            pred_labels.extend([0] * tag["count"])
+
+        for true_tag in true_tags:
+            if true_tag["name"] == tag["name"]:
+                if true_tag["count"] == tag["count"]:
+                    true_labels.extend([1] * tag["count"])
+                else:
+                    true_labels.extend([1] * true_tag["count"])
+                    true_labels.extend([0] * (tag["count"] - true_tag["count"]))
+                break
+        else:
+            true_labels.extend([0] * tag["count"])
+
+    return true_labels, pred_labels
+
+
+def test_endpoint(url: str, data: List[Dict[str, Any]]) -> Tuple[int, int, int]:
+    """
+    Sends requests with data and calculates classification report.
+
+    Args:
+        url (str): Endpoint
+        data (List[Dict[str, Any]]): Test dataset
+
+    Returns:
+        Tuple[int, int, int]: Number of false positives, true positives, false_negatives
+    """
+    true_labels, pred_labels = [], []
+    for sample in tqdm(data[:5], total=len(data)):
+        try:
+            response = requests.post(url, json=sample, headers={"Content-type": "application/json"})
+
+            if response.status_code != 200:
+                logging.error(f"Status code:{response.status_code}")
+
+            relation_scores = json.loads(response.content)
+
+            true_list, pred_list = create_true_pred_list(sample["tags"], relation_scores["tags"])
```

Comment:
```suggestion
            ner_scores = json.loads(response.content)

            true_list, pred_list = create_true_pred_list(sample["tags"], ner_scores["tags"])
```
Keeping the good naming 😄.